src/components/pages/room-membership-confirmation.yml

```diff
@@ -11,9 +11,12 @@ roomMembershipConfirmation:
   invalidToken:
     en: The invitation code is invalid or expired.
     de: Der Einladungs-Code ist ungültig oder abgelaufen.
-  invalidUser:
+  differentUser:
     en: The invitation was addressed to a different user.
     de: Der Einladung war an einen anderen Benutzer adressiert.
+  unconfirmedUser:
+    en: Your user account has not been confirmed yet.
+    de: Dein Benutzerkonto wurde noch nicht bestätigt.
```

Comment:
This is a good idea on top!

BTW: I see a changed yaml file but no updated `resources.json` :thinking:

Reply:
Thanks, now you can see the update too 😊